templates/my-orders.php

```diff
@@ -101,8 +144,98 @@
 
         </table>
 
+    <?php
+        $customer_orders_count = count( dokan_get_filtered_orders( [
+            'start_date' => $start_date,
+            'end_date'   => $end_date,
+            'vendor_id'  => $vendor_id,
+        ] ) );
+
+        $num_of_pages = ceil( $customer_orders_count / $limit );
+
+        $base_url  = get_permalink( $my_order_page_id = dokan_get_option( 'my_orders', 'dokan_pages' ) );
+
+        if ( $num_of_pages > 1 ) {
+            echo '<div class="pagination-wrap">';
```

Comment:
Instead of using `echo`, write these as html.
```
  if ( 1 < $num_of_pages ) : 
      $page_links = paginate_links( [
           'current'   => $page,
           'total'     => $num_of_pages,
           'base'      => $base_url . '%_%',
           'format'    => '?pagenum=%#%',
           'add_args'  => false,
           'type'      => 'array',
        ] );
        <?php
        <div class="pagination-wrap">
         .....
        </div>
        ?>
   endif;
```